docs/Getting Started/Recurring Tasks.md

```diff
@@ -70,6 +71,17 @@ You can validate that tasks understands your rule by using the `Tasks: Create or
 
 ---
 
+## Location of the new task
```

Comment:
Thanks for this.

Please also update the source file for the following page, to add a link to this new area of settings... The order of links in the file below is the order of the options in the settings page.

https://publish.obsidian.md/tasks/Getting+Started/Settings

Reply:
Thanks for the notice =) It's done =)